### DISCUSSION TITLE
Encrypt Sqlite3 Database

### DISCUSSION THREAD

**Discussion by feniljariwala82** [adonisjs/core#3016] [Help]
I am using the SQLite database for my AdonisJs application. And I am concerned about its security. I want to **encrypt my database** so that no one would get any advantage from it. There are some plugins available for it. But I want to know **how to encrypt the SQLite database in the Adonisjs Application**. Any guidance will be helpful!

**Comment by thetutlage** [comment]
Have you check if SQLite has support for encrypted databases at first place? If so, that will just solve your issue.

Also, I am not sure which sort of security you are concerned with. If security is the primary goal, then its better to opt for a database/service that has support for encryption. AWS Aurora supports it ( AFAIK )
